src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/ExtendedFileSource.cs

```diff
@@ -4,30 +4,30 @@
 
 namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects
 {
-    public class ExtendedFileSource
+    internal class ExtendedFileSource
     {
         [JsonProperty]
-        public JToken CopyOnly { get; internal set; }
+        internal JToken CopyOnly { get; set; }
```

Comment:
Let's make sure this doesn't prevent json serialization.

Reply:
Even if setter is private, Json serialization works...

Reply:
Json serializer can work with internal fields